static/sass/styles-v1.scss

```diff
@@ -169,6 +171,25 @@ fieldset {
   }
 }
 
+/// container for Vimeo
```

Comment:
Vanilla has 'Embedded media' util for this purpose - https://docs.vanillaframework.io/en/utilities/embedded-media